frontend/packages/dev-console/src/components/topology/topology-utils.ts

```diff
@@ -87,19 +90,25 @@ const createInstanceForResource = (resources: TopologyDataResources, utils?: Fun
  * @param dc resource item
  * @param cheURL che link
  */
-const createTopologyNodeData = (dc: OverviewItem, cheURL?: string): TopologyDataObject => {
+const createTopologyNodeData = (
+  dc: OverviewItem,
+  operatorBackedServiceKinds: string[],
+  cheURL?: string,
+): TopologyDataObject => {
   const { obj: deploymentConfig, current, previous, isRollingOut } = dc;
   const dcUID = _.get(deploymentConfig, 'metadata.uid');
   const deploymentsLabels = _.get(deploymentConfig, 'metadata.labels', {});
   const deploymentsAnnotations = _.get(deploymentConfig, 'metadata.annotations', {});
   const { buildConfigs } = dc;
+  const nodeResourceKind = _.get(deploymentConfig, 'metadata.ownerReferences[0].kind');
```

Comment:
Is the first `ownerReference` sufficient?

Reply:
Yes, as per this conversation here - https://coreos.slack.com/archives/CHGU4P8UU/p1571750071352200?thread_ts=1571748334.342700&cid=CHGU4P8UU.